src/main/resources/sql/bridge_healthdatarecord3.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS bridge_healthdatarecord3_<dateSuffix> (
+    id              CHAR(36)        ENCODE RAW          NOT NULL,
+    healthCode      CHAR(36)        ENCODE RAW          NOT NULL,
+    createdOn       BIGINT          ENCODE DELTA32k             ,
+    studyId         VARCHAR(50)     ENCODE BYTEDICT             ,
+    uploadId        CHAR(36)        ENCODE RAW          NOT NULL,
+    uploadDate      DATE            ENCODE RAW                  ,
+    schemaId        VARCHAR(250)    ENCODE LZO                  ,
```

Comment:
According to document, this stores at most 256 (the capacity of 1 byte) unique values compressed.  It's best for enum-type data.  I think we will soon grow out of 256 schemas once we introduce more studies.